build/Uno.WinUI.nuspec

```diff
@@ -614,8 +614,8 @@
 	<file src="uno.winui.win.targets" target="buildTransitive\uap10.0.16299\uno.winui.targets" />
 	<file src="uno.winui.runtime-replace.targets" target="buildTransitive\uap10.0.16299" />
 
-	<file src="uno.winui.win.targets" target="buildTransitive\uap10.0.17763\uno.winui.targets" />
-	<file src="uno.winui.runtime-replace.targets" target="buildTransitive\uap10.0.17763" />
```

Comment:
We need to keep this block, otherwise apps still referencing 17763 will behave very strangely.

Reply:
So does it mean there should be both a block for 17763 and for 18362?

Reply:
Exactly!